README.md

```diff
@@ -40,17 +40,26 @@ FLAGS:
     -V, --version                 Prints version information
 
 OPTIONS:
-    -s, --kaspad-address <kaspad-address>    The IP of the kaspad instance [default: 127.0.0.1]
-    -a, --mining-address <mining-address>    The Kaspa address for the miner reward
-    -t, --threads <num-threads>              Amount of miner threads to launch [default: number of logical cpus]
-    -p, --port <port>                        Kaspad port [default: Mainnet = 16111, Testnet = 16211]
+        --devfund <devfund-address>            Mine a percentage of the blocks to the Kaspa devfund [default: Off]
+        --devfund-percent <devfund-precent>    The precentage of blocks to send to the devfund [default: 0.1]
```

Comment:
Shouldn't it be 0.01 ?

Reply:
Well currently it is 10%, I can change it to 1%